src/Geocoder/Provider/Yandex.php

```diff
@@ -122,27 +122,34 @@ function ($value, $key) use (&$details) {$details[$key] = $value;}
 
             if (! empty($details['lowerCorner'])) {
                 $coordinates = explode(' ', $details['lowerCorner']);
-                $bounds['south'] = $coordinates[1];
-                $bounds['west']  = $coordinates[0];
+                $bounds['south'] = +$coordinates[1];
```

Comment:
It is not strictly necessary.

I've seen that results for bounds were returned as numeric strings instead
of floating point. With the plus sign it is casted to float, as in
latitude, longitude, and other providers.
Il 06/feb/2015 00:09 "Antoine Corcy" <notifications@github.com> ha scritto:

> In src/Geocoder/Provider/Yandex.php
> <https://github.com/geocoder-php/Geocoder/pull/398#discussion_r24209225>:
>
> > @@ -122,27 +122,34 @@ function ($value, $key) use (&$details) {$details[$key] = $value;}
> >
> >              if (! empty($details['lowerCorner'])) {
> >                  $coordinates = explode(' ', $details['lowerCorner']);
> > -                $bounds['south'] = $coordinates[1];
> > -                $bounds['west']  = $coordinates[0];
> > +                $bounds['south'] = +$coordinates[1];
>
> Why is a + necessary here ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geocoder-php/Geocoder/pull/398/files#r24209225>.
>